fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -1824,27 +1806,27 @@ ACTOR Future<Void> masterProxyServerCore(
 	}
 }
 
-ACTOR Future<Void> checkRemoved(Reference<AsyncVar<ServerDBInfo>> db, uint64_t recoveryCount, MasterProxyInterface myInterface) {
+ACTOR Future<Void> checkRemoved(Reference<AsyncVar<ServerDBInfo>> db, uint64_t recoveryCount,
+                                CommitProxyInterface myInterface) {
 	loop{
-		if (db->get().recoveryCount >= recoveryCount && !std::count(db->get().client.masterProxies.begin(), db->get().client.masterProxies.end(), myInterface)) {
+		if (db->get().recoveryCount >= recoveryCount &&
+		    !std::count(db->get().client.commitProxies.begin(), db->get().client.commitProxies.end(), myInterface)) {
 			throw worker_removed();
 		}
 		wait(db->onChange());
 	}
 }
 
-ACTOR Future<Void> masterProxyServer(
-	MasterProxyInterface proxy,
-	InitializeMasterProxyRequest req,
-	Reference<AsyncVar<ServerDBInfo>> db,
-	std::string whitelistBinPaths)
-{
+ACTOR Future<Void> commitProxyServer(CommitProxyInterface proxy, InitializeCommitProxyRequest req,
+                                     Reference<AsyncVar<ServerDBInfo>> db, std::string whitelistBinPaths) {
 	try {
-		state Future<Void> core = masterProxyServerCore(proxy, req.master, db, req.recoveryCount, req.recoveryTransactionVersion, req.firstProxy, whitelistBinPaths);
+		state Future<Void> core =
+		    commitProxyServerCore(proxy, req.master, db, req.recoveryCount, req.recoveryTransactionVersion,
+		                          req.firstProxy, whitelistBinPaths);
 		wait(core || checkRemoved(db, req.recoveryCount, proxy));
 	}
 	catch (Error& e) {
-		TraceEvent("MasterProxyTerminated", proxy.id()).error(e, true);
+		TraceEvent("CommitProxyTerminated", proxy.id()).error(e, true);
```

Comment:
Since this renaming is affecting many trace events, I think it's better to mention in the release notes about the change, so that people are aware of these changes and make adjustment for their toolings.

Reply:
Will do it in next PR which focus on documentation side.